src/main/java/ToDos.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Represents a ToDos task.
+ */
+public class ToDos extends Task {
```

Comment:
Perhaps Todo would be much better since there is only a single Todo task rather than multiple ones?